docs/contributing/testing/naming-conventions.mdx

```diff
@@ -0,0 +1,254 @@
+# ​Naming conventions
+
+:::caution
+
+The information in this document is a work in progress. Decisions need to be discussed and feedback
+needs to be collected before any of the information in here can be considered official.
+
+:::
```

Comment:
❓ Is placing this common practice? I support it being implied and therefore available for future improvement at any time, so this wouldn't need to be here. It'll be as official as this PR review will make i! 😉

If it's because of your "WIP" content below and you're not gonna merge it as-is, I get it. Could be worthwhile to leave the PR as a draft too.

Reply:
I initially wanted this PR to be a pure migration of the same document in confluence, but you have a good point and I think this should be removed!